txpool/txpool.go

```diff
@@ -25,12 +23,14 @@ const (
 	defaultIdlePeriod = 1 * time.Minute
 	txSlotSize        = 32 * 1024  // 32kB
 	txMaxSize         = 128 * 1024 //128Kb
```

Comment:
What do you think about making these two configurable through flags?

Reply:
I can address this readability in a separate PR.

Reply:
Let's simply create an issue for addressing this in the future (and investigate why do we enforce such a limit, when I think Ethereum doesn't).

Reply:
@dbrajovic